### PR TITLE
Fixes #9820: Stop jetty before upgrading the webapp

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9820

### CODE REVIEW

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 1]
```diff
@@ -464,13 +465,7 @@ if [ -d "%{rudderdir}/jetty7/contexts" ]; then
   ln -sf %{rudderdir}/share/webapps/rudder.xml %{rudderdir}/jetty7/contexts/rudder.xml
 fi
 
-# Warn the user that Jetty needs restarting. This can't be done automatically due to a bug in Jetty's init script.
```
Comment (amousset) [inline rudder-webapp/SPECS/rudder-webapp.spec]: Do you know what this bug was?

Comment (ncharles) [inline rudder-webapp/SPECS/rudder-webapp.spec]: it was most likely something in Jetty 7.2.2 where it didn't correctly finish/closed tty

Comment (ncharles) [inline rudder-webapp/SPECS/rudder-webapp.spec]: cannot find an upstream bug

File: rudder-webapp/debian/postinst [hunk 2]
```diff
@@ -136,11 +136,7 @@ case "$1" in
 
   # Restart the webapp
   echo -n "INFO: Restarting Rudder webapp and inventory-endpoint..."
-  if [ /etc/init.d/rudder-jetty check >/dev/null 2>&1 ];then
-    /etc/init.d/rudder-jetty start >/dev/null 2>&1
-  else
-    /etc/init.d/rudder-jetty restart >/dev/null 2>&1 || echo -n "WARNING: Jetty failed to start, maybe there is not enough RAM or swap on the machine. Skipping... "
-  fi
+  service rudder-jetty start >/dev/null 2>&1
```
Comment (ncharles) [inline rudder-webapp/debian/postinst]: here, if agent started during upgrade, then rudder-jetty is started, but is won't do anything

**Pull request merged**